Cargo.lock

```diff
@@ -2413,7 +2395,7 @@ dependencies = [
 [[package]]
 name = "zcash_encoding"
 version = "0.2.0"
-source = "git+https://github.com/zcash/librustzcash.git?rev=36d7222685e65cd8b6335f67dfe619e1a8f13197#36d7222685e65cd8b6335f67dfe619e1a8f13197"
+source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
I'm a bit hazy on how this is resolved. Does it use the version that was published as `zcash_encoding` 2.0 (which is earlier than 36d7222685e65cd8b6335f67dfe619e1a8f13197), or does it find a version of librustzcash that satisfies all the requested dependencies?